[Outlook] (manifest) update JSON support info (#3711)
OfficeDev/office-js-docs-pr: docs/outlook/smart-alerts-onmessagesend-walkthrough.md

While Smart Alerts and the [on-send feature](outlook-on-send-addins.md) provide your users the opportunity to improve their messages and meeting invites before they're sent, Smart Alerts is a newer feature that offers you more flexibility with how you prompt your users for further action. Key differences between the two features are outlined in the following table.

> [!IMPORTANT]

> > Smart Alerts aren't yet supported for the Teams manifest (preview). We are working on providing that support soon.

Reviewer: ```suggestion
> Smart Alerts aren't yet supported for the Teams manifest (preview). We're working on providing that support soon.
```
